docs/content/about/code-of-conduct-1.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Code of Conduct - I
+date: 2021-05-22
+lastmod: 2021-05-22
+---
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as organisers, attendants, students and instructors pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behaviour that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the school and the community
+* Showing empathy towards others
+
+Examples of unacceptable behaviour by participants include:
+
+* The use of sexualised language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional and academic setting
+
+
+*...continue in next column...*
```

Comment:
where are there 2 columns?

Reply:
there is a `code-of-conduct-2.md`

Reply:
Hi @vuillaut,

They are in the "back" where the links and social media links...

I divided into two because one column disturbs the website: one column is too long, so the height of the page is unnecessarily long, leaving a big gap when looking at the classes and other section of the website.

As I added them now, it will looks like this below:

<img width="1428" alt="Screenshot 2021-05-25 at 10 27 19" src="https://user-images.githubusercontent.com/8846121/119465329-dd687100-bd43-11eb-8232-0476601b1823.png">

Reply:
And I just realise I have the INDICO twice. I go to remove one :)